artwork/metalk8s-logo-vertical.svg

```diff
@@ -1,56 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>

```

Comment:
What happened here? The end result is not even a valid SVG file anymore?

Reply:
I edited in some web-based tool, will look into it. What's making you say it's not valid SVG?

Reply:
It's not even an XML file anymore if I read the diff right.

Also, why edit in whatever tool? As in, what needed to be changed? Our logo has been like this since... forever.

Reply:
Reverted